app/controllers/registrations_controller.rb

```diff
@@ -3,7 +3,8 @@ class RegistrationsController < Devise::RegistrationsController
   private
 
   def sign_up_params
-    params.require(:user).permit(:name, :email, :password, :password_confirmation)
+    password = SecureRandom.hex(10)
```

Comment:
I am not creating users to login, I just want to add them to the database (but I need one admin). Before I had hidden fields with the password in the `simple_for,` I think that this approach looks better.